client/my-sites/plans/current-plan/header.jsx

```diff
@@ -54,7 +55,15 @@ export class CurrentPlanHeader extends Component {
 					</span>
 					{ currentPlan.userIsOwner && Boolean( currentPlan.id ) && siteSlug && (
 						<Button compact href={ managePurchase( siteSlug, currentPlan.id ) }>
-							{ hasAutoRenew ? translate( 'Manage Payment' ) : translate( 'Renew Now' ) }
+							{ hasAutoRenew && translate( 'Manage Payment' ) }
+							{ ! hasAutoRenew &&
+								! shouldAddPaymentSourceInsteadOfRenewingNow(
+									currentPlan.userFacingExpiryMoment
+								) &&
+								translate( 'Renew Now' ) }
+							{ ! hasAutoRenew &&
+								shouldAddPaymentSourceInsteadOfRenewingNow( currentPlan.userFacingExpiryMoment ) &&
+								translate( 'Enable Auto Renew' ) }
```

Comment:
I don't think "Enable Auto Renew" makes sense here yet, since there's no way for users to enable auto-renewal (only after https://github.com/Automattic/wp-calypso/pull/33014 goes live in production; currently that toggle is only enabled in the development environment).

I would probably just use the default "Manage Payment" text in this case.  (Alternatively, I guess you could conditionally check for the `autorenewal-toggle` feature flag and show "Enable Auto Renew" once that is turned on, and "Manage Payment" until then.  But I think "Manage Payment" is probably fine either way - we really just want to get people to click through to the Purchase page and from there they can take specific actions to manage the subscription.)

Reply:
Done